test/cookbooks/config-driven-helper-test/recipes/mysql-server.rb

```diff
@@ -8,15 +8,15 @@
 
 if el_version >= 7
   node.set['mysql']['version'] = '5.6'
-  node.set['mysql']['server']['packages'] = ['Percona-Server-server-56', 'Percona-Server-devel-56']
+  node.set['mysql']['server']['packages-additional'] = ['Percona-Server-server-56', 'Percona-Server-devel-56']
   node.set['mysql']['server']['slow_query_log']       = 1
   node.set['mysql']['server']['slow_query_log_file']  = '/var/log/mysql/slow.log'
   node.set['mysql']['server']['service_name'] = 'mysql'
-  node.set['mysql']['client']['packages'] = ['Percona-Server-client-56', 'Percona-Server-devel-56']
+  node.set['mysql']['client']['packages-additional'] = ['Percona-Server-client-56', 'Percona-Server-devel-56']
```

Comment:
this will need changing to:

node.set['mysql']['client']['packages-additional'] = {
  'Percona-Server-client-56' => 'install',
  'Percona-Server-devel-56' => 'install',
}